lib/sig-formatter.js

```diff
@@ -0,0 +1,168 @@
+'use babel';
+
+export default class SigFormatter {
+
+  formatTypeSignatureArrows(sig) {
```

Comment:
given that this is "private" (no external production usages), could it be put down below?

Reply:
Absolutely.